thirdparty/build-thirdparty.sh

```diff
@@ -1545,12 +1545,43 @@ build_concurrentqueue() {
     cp ./*.h "${TP_INSTALL_DIR}/include/"
 }
 
+#clucene
+build_clucene() {
+    if [[ -z ${USE_AVX2} ]]; then
+        USE_AVX2=1
+    fi
```

Comment:
Is the variable needed in CLucene?

Reply:
Yes, we use TurboPFor for integer compression which need AVX2 compile flags to seperate different building arch

Reply:
Okey.